packages/common/src/services/dictionaryService.ts

```diff
@@ -27,7 +29,7 @@ const dictionaryService = (dependencies: Dependencies) => {
 				dictionary.schemas,
 			);
 
-			//TODO: Update the category to have this dictionary as current
+			await categoryRepo.updateCurrentDictionaryOnCategory(savedDictionary.id, savedCategory.id);
```

Comment:
This update was left in TO DO in previous commits but implemented until now. This is not related to this PR